changelog.d/307.bugfix

```diff
@@ -0,0 +1 @@
+Application tabs are not updated correctly #307
```

Comment:
Reference to the issue is not necessary anymore as it is already in the filename and will be automatically added by towncrier